recipes/libcf/build.sh

```diff
@@ -0,0 +1,15 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC"
```

Comment:
Please change these to: `export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC $CFLAGS"`
So the original flags are preserved.